### ISSUE TITLE
Node: Optimize variant annotation query 

### ISSUE THREAD

**Issue by hannahle** [ccmbioinfo/osmp#166]
Currently, we are querying whole gene regions using CADD and gnomAD, leading to high query time and max buffer overflow for gene size greater than 600mb, such as DMD. 

We can improve the algorithm by performing queries in sequence instead of in parallel. After retrieving the variants from multiple nodes, we find the start and end position of of the variants returned, and only query for those genomic regions. 

The algorithm can be further optimized by chunking up the regions into smaller regions, and hitting the CADD endpoint with simultaneous queries. From initial testing, it seems like CADD has a rate limit of 10 queries/ second.

**Comment by hannahle** [comment]
Closing this issue since it's being tracked in Jenny's new issue instead (#206)
